py-polars/tests/unit/operations/namespaces/files/test_show_graph.txt

```diff
@@ -0,0 +1,17 @@
+graph {
```

Comment:
Can this file be removed? I don't think we should check for the exact values as that will change over time and I don't want to recreate this file when that happens.

Reply:
Removed as requested.